spec/services/attribute_asserter_spec.rb

```diff
@@ -3,7 +3,6 @@
 RSpec.describe AttributeAsserter do
   include SamlAuthHelper
 
-  let(:ial1_user) { create(:user, :fully_registered) }
```

Comment:
(non-blocking): I like removing this and just overriding the `:user` variable at the context level, but I wonder: will this change reduce the "searchability" of tests that involve an unverified user?

Reply:
it's a good question -- for me personally i think the describe/context block naming helps with this. does that seem robust enough for you? i'm generally not in favor of adding things at the top of a big spec file that aren't used in most if not all of the tests but certainly it can have utility

Reply:
@lmgeorge i am going to merge, but if you're still unsure about this change, let's chat about it in standup -- i am going to have to open another PR against this file soon for an unrelated thing, and can always add it back in.